### DISCUSSION TITLE
Budibase table group by more conditions

### DISCUSSION THREAD

**Discussion by GarryXian** [Budibase/budibase#3426] [Help] [answered]
Hi Forks,

I wonder if budibase table can support to group by more than 1 condition?

as I want to create view from budibase table group by 2 column of the table

**Comment by GarryXian** [comment]
Hi I have found that we can customize query sql using MapReduce queries, but I just cant find the place where I can write custom query to createView。。。。
![image](https://user-images.githubusercontent.com/44110688/142395758-c769e984-b239-423e-b50e-b289096634e2.png)

**Comment by mjashanks** [comment, marked as answer]
Hi Garry.

It's not possible to create a view with multiple groups. You might be interested in this: https://github.com/Budibase/budibase/discussions/2999 - it's a collection view limitations and things that we might like views to do in future.

The "query functionality"  that your screenshot refers to is the user interface that you use when adding filters and using calculations. There is no text-based query, if that's what you are asking.
